Devscord.DiscordFramework/Delegates.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Threading.Tasks;
+using Devscord.DiscordFramework.Framework;
+using Devscord.DiscordFramework.Middlewares.Contexts;
+using Devscord.DiscordFramework.Middlewares.Factories;
+using Discord.WebSocket;
+
+namespace Devscord.DiscordFramework
+{
+    public abstract class Delegates
```

Comment:
ta nazwa Delegates nic nie mówi 
plus myśle że to mogłoby być normalnym elementem workflow